ballerina/tests/secure_client_test.bal

```diff
@@ -14,15 +14,22 @@
 // specific language governing permissions and limitations
 // under the License.
 
-import ballerina/test;
 import ballerina/jballerina.java;
 import ballerina/io;
+import ballerina/lang.runtime;
+import ballerina/test;
 
 @test:BeforeSuite
 function setupServer() returns error? {
     check startSecureServer();
 }
 
+@test:BeforeSuite
+function waitForServerWarmUp() returns error? {
+    // Wait SecureServer and EchoServer to start
+    runtime:sleep(5);
```

Comment:
Does this gurantee the server is ready? Maybe we can ping the server to check whether it is ready, and introduce a timeout to fail the test if the server is not starting within a given time.

Reply:
+1, will introduce ping after sleep

Reply:
fixed [4af7e0b](https://github.com/ballerina-platform/module-ballerina-tcp/pull/1020/commits/4af7e0b4f438f47cb65d09bc3f52f2aac02a719a)